src/components/CasesPage/CasesPage.jsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import cx from 'classnames';
+import { createBemBlockBuilder } from '../../utils';
+import { SubscriptionBanner } from '../LandingPage/SubscriptionBanner/SubscriptionBanner';
+import './CasesPage.scss';
+import downloadIcon from './icons/download.svg';
```

Comment:
```suggestion
import React from 'react';
import cx from 'classnames';

import { createBemBlockBuilder } from '../../utils';
import { SubscriptionBanner } from '../LandingPage/SubscriptionBanner/SubscriptionBanner';
import downloadIcon from './icons/download.svg';

import './CasesPage.scss';
```
Please follow this import convention to divide by empty line vendor, local and styles imports